openassessment/xblock/static/js/src/studio/oa_tiny_mce.js

```diff
@@ -23,30 +23,46 @@ export const oaTinyMCE = (options) => {
 
   const getFonts = () => CUSTOM_FONTS + STANDARD_FONTS;
   const baseAssetUrl = options.base_asset_url;
+  const staticUrl = '/static';
+
+  const sanitizeContent = (content) => {
+    // rewrite anchor link
+    let newContent = rewriteStaticLinks(
+      content,
+      `<a href="${staticUrl}`,
+      `<a href="${baseAssetUrl}`,
+    );
+
+    // rewrite image src
+    newContent = rewriteStaticLinks(
+      newContent,
+      `<img src="${staticUrl}`,
+      `<img src="${baseAssetUrl}`,
```

Comment:
This is a bit of a hack, which would fail on html like, say `<img class="w-50" src="..." />`

Why not use this the same way we do elsewhere in this library? https://github.com/openedx/edx-ora2/blob/dd6adeaf0572ecff3491e487f1df8eb0e1595c6f/openassessment/xblock/static/js/src/studio/oa_tiny_mce.js#L42

Reply:
getContent is actually get everything in html. If the user put anything even text that has `/static/` will get replace. It can be quite a confusing experience. This is just an improvement to what it was.

For image tag, it always generate `<img src="....." alt .... />`. I tested with a few iteration of input.

Reply:
Ok, so it sounds like we only support the rewriting of static links if folks insert the image via the WYSIWYG widget for adding images, and we don't rewrite static links if the user types html manually. Is that right?

If I have that right, I think a comment explaining what you've explained here would be helpful. (I would also urge you to make the comment longer than you'd consider necessary.)